lib/setup.js

```diff
@@ -42,23 +43,36 @@ import config from '../config'
 const {
   API_REQUEST_TIMEOUT,
   API_URL,
+  APP_TIMEOUT,
   COOKIE_DOMAIN,
   DEFAULT_CACHE_TIME,
   IP_BLACKLIST,
   NODE_ENV,
   OPENREDIS_URL,
   REQUEST_EXPIRE_MS,
   REQUEST_LIMIT,
+  SENTRY_PRIVATE_DSN_STAGING,
+  SENTRY_PRIVATE_DSN_PRODUCTION,
   SESSION_COOKIE_KEY,
   SESSION_COOKIE_MAX_AGE,
-  SESSION_SECRET,
-  APP_TIMEOUT
+  SESSION_SECRET
 } = config
 
 export default function (app) {
   // Timeout middleware
   if (NODE_ENV === 'production') app.use(timeout(APP_TIMEOUT || '29s'))
 
+  // Error reporting
+  let ravenDSN
```

Comment:
The reason why I avoided the ternary is that it allows for open-ended conditions -- `staging`, and then everything else; the strict logic keeps things more controlled and specific  without unintended side-effects.

But yeah, will do with `SENTRY_PRIVATE_DSN`! That makes way more sense. I copied the current form from the work I did in Prediction but I'll go back and update. This falls in line with a mistake I discovered (and fixed) on Friday related to ENV vars and our build: https://github.com/artsy/prediction/pull/422